AdobeStockImageAdminUi/Controller/Adminhtml/System/Config/TestConnection.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\AdobeStockImageAdminUi\Controller\Adminhtml\System\Config;
+
+use Exception;
+use Magento\AdobeStockImageApi\Api\GetImageListInterface;
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Api\Search\SearchCriteriaBuilder;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\Result\Json;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NotFoundException;
+use Magento\Framework\Filter\StripTags;
+
+class TestConnection extends Action
+{
+    /**
+     * Authorization level of a basic admin session.
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'Magento_AdobeStockImageAdminUi::adobe_stock_integration';
```

Comment:
I would suggest to create only one ACL resource for now. To match other configuration specific ACLs - it's would be good to configure it the following way:

        <resources>
            <resource id="Magento_Backend::admin">
                <resource id="Magento_Backend::stores">
                    <resource id="Magento_Backend::stores_settings">
                        <resource id="Magento_Config::config">
                            <resource id="Magento_AdobeStockAsset::config" title="Adobe Stock Section" translate="title" />
                        </resource>
                    </resource>
                </resource>
            </resource>
        </resources>